### DISCUSSION TITLE
Объявления проекта GyverPanelWiFi

### DISCUSSION THREAD

**Discussion by vvip-68** [vvip-68/GyverPanelWiFi#194] [Объявления]
## 👋 Начнем!
  Здесь предполагается размещать объявления связанные с развитием проекта,

**Comment by sashakozlov86** [comment]
Добрый день! Имеется ли в последней прошивке следующий функционал (если нет, предлагаю добавить)?
1. Возможность подключения микрофона с эффектами типа частотный анализатор (то есть в зависимости от шума, звука матрица обрисовывает что-то типа эквалайзера)?
2. Управление энкодером (например крутить без нажатой кнопки - менять яркость, с нажатой кнопкой - менять эффект, нажатие кнопки - вкл/выкл)?
3. Возможность озвучивания эффектов dfplaer'ом в зависимости привязки папок/фалов мп3 к эффектам?
Спасибо!

**Reply by vvip-68** [reply]
Этого нет и никогда не будет. Это, насколько я помню есть в других аналогичных проектах - используйте их.

**Reply by sashakozlov86** [reply]
Очень жаль. Да, в аналогичных проектах это есть, но их разработка на данный момент прекращена и неизвестно будет ли продолжена. В любом случае, Ваш проект очень интересен. Спасибо за его разработку и за ответ!

**Reply by kostyamat** [reply]
Никто разработку проекта FireLamp-EmbUI не прекращал. Наоборот, последнее время были очень неплохие долелки, например Gif-плеер (точнее гифки, перегнанные в формат 565, или 332). Слелан рефакторинг кода, состоялся релиз.
То, что проект потерял "русскоязычность" - дело другое. :) Винить кого знаете?

Ну и другие проекты не остановлены, кроме одного, где автор потерял сестру (убило), компьютер и желание что либо делать, после того как по  его родному Харькову ВС РФ начали методично фигачить ракетами.

**Comment by KozodoyMS** [comment]
Добрый день! Можете ли вы добавить в функционал возможность подключаться к серверам, содержащим даты праздников, чтобы выводить текст с названием праздника в нужный день?

**Reply by vvip-68** [reply]
Нет. Мне это не интересно. Код открыт и опубликован - можете сделать это сами. Или найти разработчика, который возьмется за вашу задачу.
